doc/sailfish-webview-webview.qdoc

```diff
@@ -66,6 +68,505 @@
+  \brief Allows the web view to receive programmatically defined
+         mouse events.
+
+  If \l{useQmlMouse} is true, when a mouse or touch event is received, the
+  default \c{QQuickItem} mouse or touch event handler will be invoked.
+  This means that the web view will not perform custom gesture detection.
+
+  Otherwise, the web view's \l{recvMouseMove()}, \l{recvMousePress()}
+  and \l{recvMouseRelease()} slots will be invoked automatically when the
+  respective mouse move, mouse press, and mouse release events are received
+  (and subsequently accepted) by the web view.
+
+  // TODO: no notify signal?
+
+  \sa synthTouchBegin(), synthTouchMove(), synthTouchEnd()
+  \sa recvMouseMove(), recvMousePress(), recvMouseRelease()
+*/
+
+/*!
+  \qmlproperty bool WebView::dragging
+  \readonly
+  \brief Whether the user is currently dragging the scrollable area
+*/
+
+/*!
+  \qmlproperty bool WebView::moving
+  \readonly
+  \brief Whether the scrollable area is currently moving due to
+         inertia from a previous drag gesture
+*/
+
+/*!
+  \qmlproperty bool WebView::pinching
+  \readonly
+  \brief Whether the user is currently performing a pinch gesture
+         in order to zoom the web view.
+*/
+
+/*!
+  \qmlproperty QmlMozScrollDecorator WebView::verticalScrollDecorator
+  \readonly
+  \brief The vertical scroll bar
+
+  The QmlMozScrollDecorator type is available from the
+  Qt5Mozilla import namespace.
+*/
+
+/*!
+  \qmlproperty QmlMozScrollDecorator WebView::horizontalScrollDecorator
+  \readonly
+  \brief The horizontal scroll bar
+
+  The QmlMozScrollDecorator type is available from the
+  Qt5Mozilla import namespace.
+*/
+
+/*!
+  \qmlproperty bool WebView::chrome
+  \brief Whether chrome (navigation user interface elements) are visible
+
+  By default, the chrome will be shown or hidden in response to certain
+  gestures.  The client can override this behavior by setting the value
+  manually.
+
+  \sa chromeGestureEnabled, chromeGestureThreshold
+*/
+
+/*!
+  \qmlproperty bool WebView::chromeGestureEnabled
+  \brief Whether the gesture to show the navigation chrome is enabled
+
+  If true, chrome is set to false (hiding the navigation chrome) when
+  a pan or flick gesture causes movement exceeding the chromeGestureThreshold,
+  and automatically set back to true when a similar gesture in the opposite
+  direction is detected.
+
+  \sa chrome, chromeGestureThreshold
+*/
+
+/*!
+  \qmlproperty qreal WebView::chromeGestureThreshold
+  \brief Whether the gesture to show the navigation chrome is enabled
+
+  The movement threshold for detecting a chrome gesture.
+
+  \sa chrome, chromeGestureEnabled
+*/
+
+/*!
+  \qmlproperty QMozSecurity WebView::security
+  \readonly
+  \brief Security information associated with the currently loaded page
+
+  The QMozSecurity type is available in the Qt5Mozilla import namespace.
+*/
+
+/*!
+  \qmlproperty bool WebView::desktopMode
+  \brief Whether the web view is in desktop mode
```

Comment:
Same as on Android Firefox menu "Request Desktop mode"... it more like a request desktop mode method I think and if the request cannot be served, then you continue with the previously loaded content.

Reply:
thanks, added to doc